ament_lint_common/ament_lint_common-extras.cmake.in

```diff
@@ -0,0 +1,6 @@
+# generated from ament_lint_common/ament_lint_common-extras.cmake.in
+
+set(_exec_depends "@ament_lint_common_EXEC_DEPENDS@")
+foreach(_dep ${_exec_depends})
+  find_package(${_dep} REQUIRED)
+endforeach()
```

Comment:
What's the purpose of doing this rather than exporting all of the exec depends of `ament_lint_common` with the `ament_package(DEPENDS ...)` mechanism?